src/ensembl/src/content/app/browser/drawer/Drawer.scss

```diff
@@ -14,30 +16,38 @@
     text-align: center;
   }
 
-  dd {
-    width: 600px;
+  p {
+    margin: 0px;
   }
 
-  label {
-    float: left;
-    font-weight: $light;
-    text-align: right;
-    width: 150px;
+  .nextLine {
+    display: block;
+    margin-bottom: 10px;
   }
 
-  p {
-    margin: 0 0 20px 0;
+  .clearFix {
```

Comment:
We should rename this class name to something more meaningful. "Clearfix" is a scary name from the past, when developers used floats and made sure the float will not continue to the next line by using the technique they called the "clearfix".

In our case the element with this class is more like a "grid" or a "container" or a "table", or something like that.